.github/workflows/preview.yml

```diff
@@ -126,9 +131,15 @@ jobs:
           echo >>pr-comment.txt "#### Terraform Plan"
           echo >>pr-comment.txt
           echo >>pr-comment.txt '```terraform'
-          terramate run -C stacks --changed -- terraform show -no-color out.tfplan |& dd bs=1024 count=248 >>pr-comment.txt
+          terramate run -C stacks/terraform --changed -- terraform show -no-color out.tfplan |& dd bs=1024 count=248 >>pr-comment.txt
```

Comment:
is this correct? because list runs in `stacks` dir but run will only run inside `stacks/terraform` then it will skip detected changed stacks..